tutorials/plot_sensors_decoding.py

```diff
@@ -1,12 +1,29 @@
-"""
-=================================
-Decoding sensor space data (MVPA)
-=================================
+r"""
+===============
+Decoding (MVPA)
+===============
+
+.. contents:: Contents
+   :local:
+   :depth: 3
+
+.. include:: ../git_links.inc
+
+Design philosophy
+=================
+Decoding (a.k.a. MVPA) in MNE largely follows the machine
+learning API of the scikit-learn package.
+Each estimator implements ``fit``, ``transform``, ``fit_transform``, and
+(optionally) ``inverse_transform`` methods. For more details on this design,
+visit scikit-learn_.
+
+For ease of comprehension, we will denote instantiations of the class using
+the same name as the class but in small caps instead of camel cases.
```

Comment:
Can you insert a ref here to our preprint:

https://hal.archives-ouvertes.fr/hal-01848442/

@kingjr what would be a full adequate ref?

Reply:
I'll just put "in press" for now